src/core/onerror.js

```diff
@@ -17,14 +16,6 @@ export default function onError( error, ...args ) {
 			error.stacktrace || error.fileName + ":" + error.lineNumber,
 			...args
 		);
-	} else {
-		test( "global failure", extend( function() {
-			pushFailure(
-				error.message,
-				error.stacktrace || error.fileName + ":" + error.lineNumber,
-				...args
-			);
-		}, { validTest: true } ) );
```

Comment:
I have to self-flag this, after some more thorough bashing. Here's my counter-example:
```js
QUnit.module( "foo", function() {
    QUnit.test('hello', assert => {
        assert.ok(1);
    });
    throw new Error("oops");
} );
```

In the current release, that shows a console error with `Uncaught Error: oops`, but it also shows a "global failure" as `script error`. In this PR, you see the console error, but otherwise a passing test in the report. That is a no-go, and I will re-work this.